client/src/main/kotlin/tech/figure/classification/asset/client/client/impl/DefaultACExecutor.kt

```diff
@@ -168,7 +170,17 @@ class DefaultACExecutor(
             mode = options.broadcastMode,
         ).also { response ->
             if (response.txResponse.code != 0) {
-                throw IllegalStateException("Asset classification contract execution failed with message:${System.lineSeparator()}${response.txResponse.rawLog}")
+                throw IllegalStateException(
```

Comment:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/check.html

Reply:
Agreed - `check(code == 0)` would be :dank: here

Reply:
done